### PR TITLE
zh trans update-daemon-set.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+<!--
+* The DaemonSet rolling update feature is only supported in Kubernetes version 1.6 or later.
+--->
+* Kubernetes 1.6 或者更高版本中才支持 DaemonSet 滚动更新功能。
+
```
Comment (stewart-yu) [inline content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: DaemonSet 滚动更新功能需要Kubernetes 1.6 或者更高版本支持

Comment (tanjunchen) [reply content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: DaemonSet 滚动更新功能需要 Kubernetes 1.6 或者更高版本支持。
Kubernetes 1.6 或者更高版本中才支持 DaemonSet 滚动更新功能。这个翻译个人觉得也可以。

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+  DaemonSet pods will *only* be created when you manually delete old DaemonSet
+  pods. This is the same behavior of DaemonSet in Kubernetes version 1.5 or
+  before.
+* RollingUpdate: This is the default update strategy.  
+  With `RollingUpdate` update strategy, after you update a
+  DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
+  will be created automatically, in a controlled fashion.
+--->
+* OnDelete:  使用 `OnDelete` 更新策略时，在更新 DaemonSet 模板后，只有当您手动删除老的 DaemonSet pods 之后，新的 DaemonSet pods *才会*被自动创建。跟 Kubernetes 1.6 以前的版本类似。
+* RollingUpdate: 这是默认的更新策略。使用 `RollingUpdate` 更新策略时，在更新 DaemonSet 模板后，老的 DaemonSet pods 将被终止，并且将以受控方式自动创建新的 DaemonSet pods。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: Kubernetes version 1.5 52行    Kubernetes 1.6  59行 有问题 两者没有对应上

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+{{< note >}}
+This will cause service disruption when deleted pods are not controlled by any controllers or pods are not
+replicated. This does not respect [PodDisruptionBudget](/docs/tasks/configure-pod-container/configure-pod-disruption-budget/)
+either.
+{{< /note >}}
+--->
+{{< note >}}
+当所删除的 pods 不受任何控制器管理，也不是多副本的 pods，上述操作将导致服务中断。
+同时，上述操作也不会考虑 [PodDisruptionBudget](/docs/tasks/configure-pod-container/configure-pod-disruption-budget/) 所施加的约束。
+
```
Comment (tanjunchen) [inline content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: 多余空行 建议删除

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+{{< /note >}}
+
+<!--
+#### Broken rollout
+--->
+#### 滚动中断
```
Comment (tanjunchen) [inline content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: 与下文同步 滚动更新中断

**Comment by tanjunchen** [comment]
@congfairy: PR needs rebase.

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md (hunk 1 above)
Comment (markthink) [reply content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: 我也觉得没有问题

**Comment by markthink** [comment]
@congfairy 冲突需要解决一下

**Pull request merged**